compact/hcal/lfhcal_with_space_for_insert.xml

```diff
@@ -1,18 +0,0 @@
-<lccdd>
-
-  <info name="lfhcal_with_space_for_insert.xml"
-        title="EPIC LFHCAL with insert cutout"
-        author="@FriederikeBock"
-        url="https://github.com/FriederikeBock"
-        status="development"
-        version="1.0"
-  ><comment/></info>
-
-  <define>
-    <constant name="LFHCAL_with_space_for_insert" value="1"/>
-    <constant name="LFHCAL_without_space_for_insert" value="0"/>
```

Comment:
Do we need to hardcode these to lfhcal.xml now?

Reply:
Any use of these was already removed in #798, see `lfhcal.xml`. They were used in `<position x= y= z= if="LFHCAL_with_space_for_insert"/>` placement statements. None of those are left.

Reply:
See a9d0cb3